client/packages/cli/src/index.js

```diff
@@ -1040,7 +1040,7 @@ function linkOptsPretty(attr) {
 async function pushSchema(appId, opts) {
   const res = await readLocalSchemaFileWithErrorLogging();
   if (!res) return { ok: false };
-  const { schema } = res;
+  var { schema } = res;
```

Comment:
Nit: intentional?

Reply:
Damn. An artifact of rolling back original impl